src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
```

Comment:
Ensure that the variable `$main-text-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.